flang/lib/Lower/IntrinsicCall.cpp

```diff
@@ -469,7 +469,10 @@ struct IntrinsicLibrary {
   mlir::Value genDprod(mlir::Type, llvm::ArrayRef<mlir::Value>);
   template <Extremum, ExtremumBehavior>
   mlir::Value genExtremum(mlir::Type, llvm::ArrayRef<mlir::Value>);
+  mlir::Value genExponent(mlir::Type, llvm::ArrayRef<mlir::Value>);
```

Comment:
nit: `genExponent` should come before `genExtremum` to respect the alphabetical order (though it has no functional consequences here).

Reply:
Yes I will change this. Thank you!